build/package/i2b2/docker-entrypoint.sh

```diff
@@ -26,7 +26,7 @@ EOSQL
     done
 
     # delete loaded data
-    rm -rf "$I2B2_DATA_DIR"/{*,.*} || true
+    rm -rf "$I2B2_DATA_DIR"/.git "$I2B2_DATA_DIR"/*
```

Comment:
with this modification the script fails, doesn't it?

Reply:
Why would it fail? 
(that's also why i've put the PR as draft - to wait for the CI to validate it works)

Reply:
Do you refer at removing the "true"? That's actually why I made the change. Adding the "true" was a dirty hack to make the line work more than anything. It would try to remove `.` and `..` folders which would fail, but still remove the other stuff we wanted gone. Now it's cleaner.

Reply:
yep yep, you're right. one weird thing, I looked at the log of the CI, and it seems like it still tries to remove `.` and `..`, so it should still throw the error, except that it does not...oh well

Reply:
Ah it does? Can you point me to the log in question? Did not see that

Reply:
lines 4855-56
```
rm: refusing to remove '.' or '..' directory: skipping '/i2b2-data/.'
rm: refusing to remove '.' or '..' directory: skipping '/i2b2-data/..'
```